paddle/operators/CMakeLists.txt

```diff
@@ -1,6 +1,44 @@
-if(WITH_GPU)
-    nv_library(add_op SRCS add_op.cc add_op.cu DEPS operator op_registry glog ddim)
-else()
-    cc_library(add_op SRCS add_op.cc DEPS operator op_registry glog ddim)
-endif()
+function(op_library TARGET)
```

Comment:
This not in `generic.cmake` because it is only used by building an operator, not a `generic function`.